src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -299,7 +299,8 @@ private void btnFileBug_Click(object sender, RoutedEventArgs e)
                 // File a new bug
                 Logger.PublishTelemetryEvent(TelemetryAction.Scan_File_Bug, new Dictionary<TelemetryProperty, string>() {
                     { TelemetryProperty.By, FileBugRequestSource.HowtoFix.ToString() },
-                    { TelemetryProperty.IsAlreadyLoggedIn, IssueReporter.IsConnected.ToString(CultureInfo.InvariantCulture) }
+                    { TelemetryProperty.IsAlreadyLoggedIn, IssueReporter.IsConnected.ToString(CultureInfo.InvariantCulture) },
```

Comment:
For the entry points that send redundant telemetry, how do folks feel about a common class for telemetry events? I am imagining an IssueFilingTelemetryEventCreator that takes the 'By' parameter and adds the IssueReporter information by itself. This is somewhat similar to the web pattern here: https://github.com/Microsoft/accessibility-insights-web/blob/9f2b53c62105967e3b3b27f05d1d27b8e0fd42ba/src/common/telemetry-data-factory.ts

We may not want to go that far in this particular PR, but flagging it FYI